modules/rostests/winetests/advapi32/registry.c

```diff
@@ -3622,25 +3622,25 @@ static void test_RegQueryValueExPerformanceData(void)
 
         cbData = 0xdeadbeef;
         dwret = RegQueryValueExA(HKEY_PERFORMANCE_TEXT, names[i], NULL, NULL, NULL, &cbData);
-todo_wine
+        todo_wine
```

Comment:
I assume those are left-aligned for a specific purpose,
and this is wine-synced code anyway so we prefer to not create diffs like this in them.